### ISSUE TITLE
Using Multipoolminer on Miningpoolhub getting status failed with Ethhash and equihash process says running

### ISSUE THREAD

**Issue by harssh** [MultiPoolMiner/MultiPoolMiner#978]
I am using multipool miner on miningpoolhub while execution I am getting some output which is little confusing for me if anyone aware of what it means I am adding a link to screenshot and this two outputs are showing up at the same time.

https://imgur.com/a/6v511
Text version

>Status: Running

Speed Active Launched Command

>0 H/s 00 Days 07 Hours 29 Minutes 3 Times Bin\Equihash->DSTM\zm.exe --telemetry --server >us-east.equihash->hub.miningpoolhub.com --port 17023

>Status: Failed

>Speed Active Launched Command

>0 H/s 00 Days 07 Hours 30 Minutes 3 Times Bin\Equihash->DSTM\zm.exe --telemetry --server >us-east.equihash->hub.miningpoolhub.com --port 17023

>0 H/s 00 Days 00 Hours 07 Minutes 2 Times Bin\Ethash->Claymore\EthDcrMiner64.exe -r -1 ->mport 23333 -epool us-east.ethash->hub.miningpoolhub.com:20585 -ewal magnum.mageth ->epsw x ->esm 3 -allpools 1 -allcoins exp -dpool >hub.miningpoolhub.com:17022 -dwal >magnum.mageth -dpsw x ->dcoin sc -dcri 30 -platform 2

Is this normal or I am missing some config ...
Using GTX 1060 6gb

**Comment by UselessGuru** [comment]
Your miner crashes.
What happens if you run the miner 'standalone' with the exact same parameters?
If the is an error please post a screenshot as well as the complete command line for each miner that crashes.

**Comment by harssh** [comment]
So problem has solved after restart . 
What I noticed was among three rigs only one did't had issue like this and was able to recollect that I did a restart after installing and benchmarking multipoolminer for miningpoolhub.

I did the same (restarted) other two machines one at a time saw this error going away too.

May be I don't know but for me my rigs need a one time restart after setting/benchmarking  multipoolminer.
